### ISSUE TITLE
Update documentation to prevent dropping tcp connections

### ISSUE THREAD

**Issue by spinscale** [elastic/elasticsearch#5067]
See mailinglist thread https://groups.google.com/d/msg/elasticsearch/hRQRLW-9zAo/Wm24krbQ8roJ

```
I found out that a firewall in the middle was dropping open connections with x amount of time of inactivity.
ES wasn't really happy with this apparently.

Solution:

set network.tcp.keep_alive true

Add following params to sysctl.conf:

net.ipv4.tcp_keepalive_time = 60
net.ipv4.tcp_keepalive_probes = 6
net.ipv4.tcp_keepalive_intvl = 10
```

I think it makes sense to at least mention this somewhere in the docs in order to help the people.

**Comment by cmoad** [comment]
I have reason to believe this issue effected us as well on Google Compute Engine, which drops idle TCP connections after 10 minutes.

https://cloud.google.com/compute/docs/troubleshooting#communicatewithinternet

Their recommended settings:
`sudo /sbin/sysctl -w net.ipv4.tcp_keepalive_time=60 net.ipv4.tcp_keepalive_intvl=60 net.ipv4.tcp_keepalive_probes=5`

**Comment by spinscale** [comment]
This should be closed by #10189
